css/index.css

```diff
@@ -1,3 +1,520 @@
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
 
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* The use of !important was avoided so Carlos wouldn't cry himself to sleep */
+
+/* 2 Stretch Tasks achieved*/
+
+/* Gotta make some rules here so that everything has a limit */
+.container {
+	width: 950px;
+	margin: auto;
+}
+
+/* Nav bar and header styles */ 
+
+header {
+	height: 60px;
+	width: 950px;
+	position: fixed;
+	top: 0;
+	background-image: linear-gradient(to bottom right, rgb(0, 110, 255), rgb(255, 255, 251));
+	z-index: 10;
+    display: flex;
+
+}
+
+header nav {
+	height: 60px;
+	width: 100%;
+	display: flex;
+	position: relative;
+	line-height: 25px;
+	flex-direction: row;
+	
+}
+
+header nav a {
+	margin: 10px 60px 0 20px;
+	text-decoration: none;
+	justify-content: flex-start;
+	color: rgb(68, 69, 160);
+	line-height: 36px;
+	font-size: 14px;
+}
+
+header .logo {
+	margin-top: 5px;
+	margin-right: 5px;
+	height: 40px;
+    width: auto;
+	justify-content: flex-start;
+}
+
+/* Hands down the most challenging part, at one point the fonts even got wacky on me */
+.main-content {
+	width: 100%;
+	height: 475px;
+	position: relative;
+	display: flex;
+	margin-top: 60px;
+}
+
+.main-content .stack-text {
+	position: absolute;
+	margin-top: 50px;
+	margin-left: 35px;
+	height: 310px;
+	width: 60%;
+	text-align: center;
+	line-height: 80px;
+	font-size: 75px;
+}
+
+.main-content .snippy {
+	position: absolute;
+	right: 0;
+	top: 10;
+	height: 380px;
+	width: 40%;
+}
+
+.main-content .start {
+	position: absolute;
+	border: 1px solid gray;
+	left: 230px;
+	bottom: 125px;
+	padding: 5px 45px 3px 45px;
+	height: 35px;
+	font-size: 18px;
+	text-align: center;
+}
+
+/* After what happened ^^^ there, this was pretty simple */
+h2 {
+	font-size: 16px;
+}
+
+.info-content .feature-about {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+.info-content .feature {
+	padding-top: 10px;
+	margin-right: 20px;
+	width: 45%;
+}
+
+.info-content .about {
+	padding-top: 10px;
+	margin-left: 20px;
+	width: 45%;
+}
+
+.middle-img {
+	margin: 35px 0;
+	width: 100%;
+}
+
+.info-content .ser-pro-vis {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+}
+
+.info-content .services {
+	margin-right: 10px;
+	width: 30%;
+	padding-bottom: 25px;
+}
+
+.info-content .product {
+	margin: 0 5px;
+	width: 30%;
+}
+
+.info-content .vision {
+	margin-left: 10px;
+	width: 30%;
+}	
+
+footer {
+	text-align: center;
+	margin-top: 100px;
+}
+
+/* And so begins the tablet break */
+
+@media (max-width: 800px){
+
+    .container {
+        width: 800px;
```

Comment:
fantastic use of the container to set the width of the rest of your elements, great job.